gluster-exporter/main.go

```diff
@@ -41,8 +46,51 @@ func registerMetric(name string, fn func(), intervalSeconds int64) {
 	glusterMetrics = append(glusterMetrics, glusterMetric{name: name, fn: fn, intervalSeconds: time.Duration(intervalSeconds)})
 }
 
+func getGlusterdWorkdir() string {
```

Comment:
This function should just return workdir

    if *glusterdWorkdir != "" {
        return *glusterdWorkdir
    }
    if *glusterMgmt == "glusterd2" {
        return defaultGlusterd2Workdir
    }
    return defaultGlusterd1Workdir